src/core/js/enums/questionButtonStateEnum.js

```diff
@@ -0,0 +1,14 @@
+define(function() {
+
+    var QUESTION_BUTTON_STATE = ENUM([
```

Comment:
`BUTTON_STATE` would not only be shorter but would also allow for inclusion of non-question button states in the future